### PR TITLE
added image and input component

### PR DESCRIPTION
**Pull request by narharikale**

### CODE REVIEW

**Review by ScoobyDubeyDo — APPROVED** [review]

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>image</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"rel="stylesheet">
```
Comment (samflab) [inline components/image/image.html:8]: add the fonts in the css file, this is helpful when you import your CL in your next project

**Review by samflab — APPROVED** [review]
Everything else looks good.

**Review by ayush65 — APPROVED** [review]
Looks good

File: components/image/image.html (hunk 1 above)
Comment (narharikale) [reply components/image/image.html:8]: hey, @samflab thanks for the suggestion.
but this is a link for google material icons

**Pull request merged**